packages/shared-configs/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@cronn/shared-configs",
+  "version": "0.0.0",
+  "description": "Internal shared configs for the monorepo",
+  "author": "cronn",
+  "private": true,
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cronn/vitest-file-snapshots.git",
+    "directory": "packages/shared-configs"
+  },
+  "homepage": "https://github.com/cronn/vitest-file-snapshots",
+  "engines": {
+    "node": ">=22",
+    "pnpm": ">=10"
+  },
+  "scripts": {
+    "check": "biome check .",
+    "check:fix": "biome check --write .",
+    "compile": "tsc",
+    "build": "tsup",
+    "ci": "biome ci . && pnpm run compile && pnpm run build"
```

Comment:
When running `pnpm run ci` in this folder, I am getting the following error:

```
> biome ci . && pnpm run compile && pnpm run build

configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Key 'insert_final_newline' is incompatible with biome: Biome always inserts a final newline. Set this option to true.
  

Checked 5 files in 1944µs. No fixes applied.

> @cronn/shared-configs@0.0.0 compile /home/philipp/Documents/ga-lotse/vitest-file-snapshots/packages/shared-configs
> tsc

sh: 1: tsc: not found
```

1. I think the biome warning is a known issue?
2. "tsc not found" is the problem.

Reply:
1. Yes, there is an [open issue](https://github.com/biomejs/biome/issues/5165) for it.
2. I could not reproduce the issue. Was your local installation incomplete or did some symlinks not work?

Reply:
Steps to reproduce:

1. On this branch run:
2. `git clean -xdf .` to clean the repo (just like new cloning)
3. `pnpm install`
4. `pnpm run --recursive ci`

node -v       
v22.16.0

pnpm -v                
10.11.0

`shared-config` has no dependency on `typescript` (and also root has no dependency on typescript). When using the "non-flat node_modules directory" with pnpm, the package can not resolve `tsc`.

Adding `typescript` to the devdependencies fixes this for me.

The question is why is this working in the pipeline and on your machine?